docker/Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: push
+
+TAG = 0.1
+PREFIX = quay.io/coffeepac/kafka-monitor
```

Comment:
Fair!  I meant to ask about this.  quay.io is a public image repo, operated by coreos.  It's what my team uses. So nothing special about it.  

Does linkedin have a normal location?  Im happy to write it there.  linkedin/kafka-monitor or something?